freqtrade/main.py

```diff
@@ -32,8 +31,6 @@ def main(sysargv: List[str] = None) -> None:
     return_code: Any = 1
     worker = None
     try:
-        set_loggers()
```

Comment:
My initial thought was that this may show uninitialized log messages, but i could not find a way to support my thought :+1:

Reply:
It's called here with verbosity set to 0 anyway.